crates/adapters/src/format/avro/output.rs

```diff
@@ -417,6 +434,12 @@ impl AvroEncoder {
         })
     }
 
+    fn value_schema(&self) -> Schema {
```

Comment:
This is problematic: we will create a deep copy of the schema for each record we output.